examples/with-react-native-web/next.config.js

```diff
@@ -0,0 +1,32 @@
+const esNodeModules = /react-native-web(?!.*node_modules)/
+const cjsNodeModules = /node_modules\/(?!react-native-web(?!.*node_modules))/
+
+module.exports = {
+  webpack: (config, { buildId, dev, isServer, defaultLoaders }) => {
+    // Alias all `react-native` imports to `react-native-web`
+    config.resolve.alias = {
+      'react-native$': 'react-native-web'
```

Comment:
Lets drop this and fix this in the code instead.
https://github.com/zeit/next.js/blob/canary/examples/with-react-native-web/pages/index.js#L2

Reply:
Making that change goes against the [primary goal](https://github.com/necolas/react-native-web#contributing) of `react-native-web`.

>The main purpose of this repository is to help evolve React web and native development towards the platform-agnostic design of React Native

The primary goal of `react-native-web` is to leverage the _platform-agnostic_ design of React Native. Having a file specifically import `react-native-web` would prevent using that file on native platforms.

I strongly recommend not removing this webpack alias. It is a part of the [standard "Getting started" guide](https://github.com/necolas/react-native-web/blob/master/docs/guides/getting-started.md#configuring-a-module-bundler) provided by `react-native-web`.